src/transformers/models/deta/modeling_deta.py

```diff
@@ -1804,12 +1811,15 @@ def __init__(self, config: DetaConfig):
         self.post_init()
 
     @torch.jit.unused
-    # Copied from transformers.models.deformable_detr.modeling_deformable_detr.DeformableDetrForObjectDetection._set_aux_loss
     def _set_aux_loss(self, outputs_class, outputs_coord):
         # this is a workaround to make torchscript happy, as torchscript
         # doesn't support dictionary with non-homogeneous values, such
         # as a dict having both a Tensor and a list.
-        return [{"logits": a, "pred_boxes": b} for a, b in zip(outputs_class[:-1], outputs_coord[:-1])]
+        aux_loss = []
+        # TO DO: prettify
```

Comment:
We shouldn't leave to-dos to make code pretty - it should be done in the original PR

FWIW - I think the code here is fine but you can do something like: 

```py
aux_loss = [
	{"logits": logits, "pred_boxes": pred_boxes} for logits, pred_boxes in zip(outputs_class.T, outputs_coord.T)
]
```

Reply:
done!